script.js

```diff
@@ -0,0 +1,223 @@
+let minTemp = document.querySelectorAll('.minTemp');
+let maxTemp = document.querySelectorAll('.maxTemp');
+let futureCastImg = document.querySelectorAll('.futureCastImg');
+const tempDetailDisplay = document.querySelector('.tempDetail');
+const dayValues = document.querySelectorAll('.dayValue');
+const celciusBtn = document.querySelector('.celciusBtn');
+const farenheitBtn = document.querySelector('.farenheitBtn');
+const todayTempDisplay = document.querySelector('.todayTemp');
+const locationBtn = document.querySelector('.LocationBtn');
+const windSpeedDisplay = document.querySelector('.windSpeed');
+const humidityDisplay = document.querySelector('.humidity');
+const inputRange = document.querySelector('.inputRange');
+const visibiltyDisplay = document.querySelector('.visibilty');
+const airPressureDisplay = document.querySelector('.airPressure');
+const inputLocation = document.querySelector('.inputLocation');
+const searchBtn = document.querySelector('.searchLocation');
+const todayWeatherImg = document.querySelector('.todayWeatherImg');
+const tempScale = document.querySelectorAll('.tempScale');
+const celciusText = document.querySelector('.celciusText');
+locationBtn.addEventListener('click', getWeatherData);
+searchBtn.addEventListener('click', searchLocation);
+const cityInfo = document.querySelector('.cityInfo');
+async function fetchData(latitude, longitude) {
+    const getUrl = await fetch(`https://api.openweathermap.org/data/2.5/forecast?lat=${latitude}&lon=${longitude}&appid=3f48274a20bac99a1e52fb12652e7fdc`, { mode: 'cors' })
+    const data = await getUrl.json();
+    return data;
+}
+async function MinTempData(data) {
+    const minTemp1 = (await data.list[8].main.temp_min - 273).toFixed(2);
+    const minTemp2 = (await data.list[16].main.temp_min - 273).toFixed(2);
+    const minTemp3 = (await data.list[24].main.temp_min - 273).toFixed(2);
+    const minTemp4 = (await data.list[30].main.temp_min - 273).toFixed(2);
+    const minTemp5 = (await data.list[36].main.temp_min - 273).toFixed(2);
+    const minTempArray = [
+        minTemp1, minTemp2, minTemp3, minTemp4, minTemp5
+    ]
+    return minTempArray;
```

Comment:
This doesn't need to be marked as async and it also doesn't need to use await since `data` is not a promise